website/docs/getting-started/getting-started-guide.md

```diff
@@ -97,19 +97,19 @@ Flags:
 Use "ksctl create-cluster [command] --help" for more information about a command.
 ```
 
-This will have some options specific to the cloud provider. Below, we'll take a look at how you can create clusters in civo.
+This will have some options specific to the cloud provider. Below, we'll take a look at how you can create clusters in respected cloud provider.
 
-Apart from using the `ksctl create civo` command, we can also insert some flags. 
+Apart from using the `ksctl create <cloud-provider>` command, we can also insert some flags. 
 Some flags which will be common across every provider are `Cluster name`, `Node size`, `Region`, and `Number of nodes`.
 
 ```bash
-ksctl create-cluster civo --help
+ksctl create-cluster <cloud-provider> --help
 It is used to create cluster with the given name from user. For example:
 
-ksctl create-cluster civo <arguments to civo cloud provider>
+ksctl create-cluster <cloud-provider> <arguments to cloud provider>
 
 Usage:
-  ksctl create-cluster civo [flags]
+  ksctl create-cluster <cloud-provider> [flags]
 
 Flags:
   -a, --apps string       PreInstalled Apps with comma seperated string
```

Comment:
we can replace this block by another block which specifies to use -h flag to get the flags and sub-commands present